docs/expansionPort.md

```diff
@@ -0,0 +1,115 @@
+# AMDC Expansion Port
+
+This document describes the design considerations and implementation details for the expansion port interface on the Advanced Motor Drive Controller (AMDC). The expansion port consists of the isoSPI and differential I/O interface. The block diagram of the implementation is presented, and each block is discussed in detail. The connector interface and PCB layout information are also provided in this document.
+
+
+## Relevant Versions of AMDC Hardware
+
+AMDC REV D
+
+
+## Design Requirements and Considerations
+
+The design requirements for the isoSPI and differential I/O (D_IO) interface are as follows:
+
+1. Enable differential communication to ensure high noise immunity and to support longer cable lengths.
+
+2. Achieve high throughput using differential I/O interface (limited by the external device and the FPGA speed). The typical propagation delay of the interface is 7ns.
+
+3. Eliminate common-mode noise using an isolation barrier for isoSPI and also provides electrical isolation for safety.
+
+
+## Block Diagram / External Connections
+
+<img src="images/amdc-isoSPI.svg" />
+
+### 1. DB15 Connector
+
+A total of two DB15 D-sub high-density connectors are used for both the isoSPI and differential I/O (D_I / D_O) interface. Each connector has pins for 2 isoSPI communication interfaces, 2 differential I/O interfaces, 1 pin for ground signals, and 1 pin for 5V supply from the AMDC. More details on the DB15 connector can be found in [here](https://content.norcomp.net/rohspdfs/Connectors/17Y/178/513/178-H15-513R497.pdf). The location of these connectors in the AMDC is shown below:
+
+<img src="images/amdc-isoSPI-input-highlighted.svg" />
+
+The pin mappings for each DB15 connector is shown below:
+
+| Pin number | Signal name |
+|------------|--------|
+| 1 | 5V |
+| 2 | D1_I_P |
+| 3 | D1_I_N |
+| 4 | D2_I_P |
+| 5 | D2_I_N |
+| 6 | NC |
+| 7 | isoSPI1_P |
+| 8 | isoSPI1_N |
+| 9 | isoSPI2_P |
+| 10 | isoSPI2_N |
+| 11 | GND |
+| 12 | D1_O_P |
+| 13 | D1_O_N |
+| 14 | D2_O_P |
+| 15 | D2_O_N |
+
+The isoSPI signals that connect to the PicoZed pins and Zynq-7000 FPGA module can be found at the isoSPI section of the [pin mapping document](RevD-PinMapping.md#encoder).
+
+### 2. IsoSPI communication interface

```

Comment:
Can we reframe this section to be more useful from a user perspective? I.e., I think the most common use case for this document is from someone who is looking to design an expansion board for the AMDC. They are going to want to know what they should do in their design to successfully interface to the IsoSPI pins. What recommendations can you give them for their design?